From: Bragatheswaran Manickavel <bragathemanick0908@gmail.com>
Date: Sun, 17 Sep 2023 14:04:43 +0530
Subject: thermal/drivers/imx8mm_thermal: Fix function pointer declaration by
 adding identifier name
Patch-mainline: v6.7-rc1
Git-commit: f84f6e0f4588650e22543376d3333d65d578ff73
References: jsc#PED-11164

Added identifier names to respective definitions for fix
warnings reported by checkpatch.pl

WARNING: function definition argument 'void *' should also have an identifier name
WARNING: function definition argument 'int *' should also have an identifier name
Signed-off-by: Bragatheswaran Manickavel <bragathemanick0908@gmail.com>
Signed-off-by: Daniel Lezcano <daniel.lezcano@linaro.org>
Link: https://lore.kernel.org/r/20230917083443.3220-1-bragathemanick0908@gmail.com
Acked-by: Chun-Yi Lee <jlee@suse.com>
---
 drivers/thermal/imx8mm_thermal.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/thermal/imx8mm_thermal.c
+++ b/drivers/thermal/imx8mm_thermal.c
@@ -79,7 +79,7 @@
 struct thermal_soc_data {
 	u32 num_sensors;
 	u32 version;
-	int (*get_temp)(void *, int *);
+	int (*get_temp)(void *data, int *temp);
 };
 
 struct tmu_sensor {
